task/sast-java-sec-check/0.1/README.md

```diff
@@ -0,0 +1,24 @@
+# sast-java-sec-check task (Skipped)
+
+## Description:
+
+sast-java-sec-check task makes use of FindSecBugs-CLI, which is a command-line interface tool for analyzing Java bytecode to identify security vulnerabilities. It is a part of the [FindSecBugs project](https://find-sec-bugs.github.io/), which is an open-source security scanner for Java applications.
+
+FindSecBugs-CLI uses a set of rules and checks to scan Java bytecode and generate a report of potential security issues. It supports various types of vulnerabilities, including SQL injection, cross-site scripting (XSS), deserialization vulnerabilities, and others.
+
+This task is currently skipped.
+
+## Params:
+
+| name                | description                                                                   |
+|---------------------|-------------------------------------------------------------------------------|
+| PATH_CONTEXT        | Path to your source code.                                                 |
+| OUTPUT_FORMAT       | Format of findsecbugs output.                                             |
+| OUTPUT_ONLY_ANALYZE | Only analyze given classes and packages; end with .* to indicate classes in a package, .- to indicate a package prefix. |
```

Comment:
```suggestion
| OUTPUT_ONLY_ANALYZE | Analyze only given classes and packages; Ending with .* to indicate classes in a package, .- to indicate a package prefix. |
```
Refer to the reviewed description in https://github.com/redhat-appstudio/build-definitions/pull/331/files#diff-73b749ccae297246f619ca11fe7763a1a7838669bb011f9b0537a860be3300e5R30.

Reply:
updated, thanks!